service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsControllerTest.java

```diff
@@ -70,28 +78,76 @@ void shouldNotReturnAnErrorWhenHearingDateIsSetInDistantFuture() throws Exceptio
         LocalDateTime distantFuture = LocalDateTime.now().plusYears(1000);
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(createHearing(
-            distantFuture, distantFuture.plusDays(1)));
+            distantFuture, distantFuture.plusDays(1)), EMPTY_BOOKING);
+
+        assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
+    }
+
+    @Test
+    void shouldOnlyValidateNewBookingsWhenBookingAlreadyExists() throws Exception {
+        LocalDateTime distantFuture = LocalDateTime.now().plusYears(1000);
+        LocalDateTime distantPast = LocalDateTime.now().minusYears(10000);
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(createHearing(
+            distantFuture, distantFuture.plusDays(1)), createHearing(distantPast, distantPast.plusDays(1)));
 
         assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
     }
 
+    @Test
+    void shouldErrorWhenExistingBookingIsUpdatedToPastDate() {
+        LocalDateTime futureDate = LocalDateTime.now().plusDays(5);
+        UUID hearingId = randomUUID();
+
+        CallbackRequest request = callbackRequestWithEditedBooking(futureDate, hearingId);
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = postMidEvent(request);
+
+        assertThat(callbackResponse.getErrors()).containsOnlyOnce(ERROR_MESSAGE);
+    }
+
+    private CallbackRequest callbackRequestWithEditedBooking(LocalDateTime futureDate, UUID hearingId) {
+        return CallbackRequest.builder()
+                .caseDetails(CaseDetails.builder()
+                    .data(Map.of("hearingDetails", listBookingWithStartDate(hearingId, futureDate.minusDays(10))))
+                    .build())
+                .caseDetailsBefore(CaseDetails.builder()
+                    .data(Map.of("hearingDetails", listBookingWithStartDate(hearingId, futureDate)))
+                    .build())
+                .build();
+    }
+
     private HearingBooking createHearing(LocalDateTime startDate, LocalDateTime endDate) {
         return HearingBooking.builder()
             .startDate(startDate)
             .endDate(endDate)
             .build();
     }
 
-    private AboutToStartOrSubmitCallbackResponse makeRequest(HearingBooking hearingDetail) throws Exception {
-        Map<String, Object> map = mapper.readValue(mapper.writeValueAsString(hearingDetail),
+    private AboutToStartOrSubmitCallbackResponse makeRequest(HearingBooking after, HearingBooking before) throws Exception {
+        Map<String, Object> mapAfter = mapper.readValue(mapper.writeValueAsString(after),
```

Comment:
really minor, naming variable after type is not best option, it does not give any context what is is